### ISSUE TITLE
Problem with Task 6

### ISSUE THREAD

**Issue by yellowjs0304** [Kelym/FAST#3]
hello.
I'm sorry to raise another issue.
I have problem with final command.
`
python tasks/R2R/run_search.py --job search
                               --load_follower tasks/R2R/experiments/smna/snapshots/[name of the latest model]
                               --max_episode_len 55
                               --K 20
                               --logit 
                               --experiment_name tmp
                               --load_reranker tasks/R2R/experiments/[name of the reranker model]
`
I ran Training reranker.ipynb and got new candidates_ranker_xxxx models.
but, after that i ran your command, got error like below.
![캡처](https://user-images.githubusercontent.com/30281608/63533256-5ae7c780-c547-11e9-8e94-8409915a66c2.PNG)

I think these caused by this line in your .ipynb file.
![캡처2](https://user-images.githubusercontent.com/30281608/63533316-7eab0d80-c547-11e9-9b09-f2104263d7a3.PNG)

I don't know why you did the input_dim size to 7. Could you please explain about this?
And how do i fix the problem?

I'll be waiting for the reply.
Thank you.

**Comment by Kelym** [comment]
The error was caused by having trained a Reranker that has 7-input-dim and trying to load the model to a 28-input-dim reranker. 
The 7-dim reranker considers 7 features for each candidate path. See in the notebook `Training Reranker.ipynb`, Cell 3: `new_data['reranker_inputs'].append([len(world_states), sum_logits, mean_logits, sum_logp, mean_logp, pm, speaker]`. 
The `run_search.py` code tried to load a 28-dim reranker because of a separate experiment we ran, where we would _duplicate_ the 7 features for each candidate path to be 28 features. i.e.`new_feature_for_path = [7 features] * 4`. 
Since our latest experiment uses 28-dim reranker, the simple fix would be to train a 28-input-dim reranker. To do so, I have edited one line in the jupyter notebook ([commit](https://github.com/Kelym/FAST/commit/1832e771d917899a7b643716c8da5f06b055ef4a)) and should solve this problem. 

If you are interested in playing with reranker feature engineering, feel free to change codes in:
1) `Training Reranker.ipynb`, `new_data['reranker_inputs'].append([feature1, feature2, feature3, ...])`
2) `run_search.py` loading reranker at line 350, use the correct dimension.
3) `follower.py` use reranker at line 967, generate the correct feature vector. 

By the way I uploaded our latest 28D reranker to the Google Drive as well. Hope this helps!
